src/common/device_helpers.cuh

```diff
@@ -866,25 +870,39 @@ class AllReducer {
    * group.
    *
    * \param device_ordinals The device ordinals.
+   * \param opg Whether to initialize AllReducer with a single process per GPU
```

Comment:
This param has been removed, right?

Reply:
Yes! Thanks @rongou for catching this stray comment. There is no longer a parameter for `opg` anywhere.